src/libraries/Common/tests/CoreFx.Private.TestUtilities/System/PlatformDetection.Windows.cs

```diff
@@ -63,6 +63,8 @@ public static partial class PlatformDetection
         // Per https://docs.microsoft.com/en-us/windows-insider/flight-hub/ the first 20H1 build is 18836.
         public static bool IsWindows10Version2004OrGreater => IsWindows &&
```

Comment:
Is this one still used anywhere, and if yes, are those correct?

Reply:
Yes, it's used in [checking for SSL3 support](https://github.com/dotnet/runtime/blob/3a769f77f6e73baa8fcc66a6cd6ebdb4611c3d55/src/libraries/Common/tests/CoreFx.Private.TestUtilities/System/PlatformDetection.cs#L201). Seems valid to me.